nntrainer/compiler/multiout_realizer.cpp

```diff
@@ -9,16 +9,99 @@
  * @author Jihoon Lee <jhoon.it.lee@samsung.com>
  * @bug No known bugs except for NYI items
  */
+#include <stdexcept>
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+#include <utility>
+
+#include <common_properties.h>
 #include <compiler_fwd.h>
+#include <layer_node.h>
 #include <multiout_realizer.h>
+#include <remap_realizer.h>
 
 namespace nntrainer {
 MultioutRealizer::~MultioutRealizer() {}
 
 GraphRepresentation
 MultioutRealizer::realize(const GraphRepresentation &reference) {
+  GraphRepresentation processed(reference.begin(), reference.end());
+
+  std::unordered_multiset<std::string> freq_map;
+  std::unordered_set<std::string> node_names;
+  std::unordered_set<std::string> connection_names;
```

Comment:
consider using `std::unordered_map<std::string, unsigned int>` as a combined struct for `freq_map` and `connection_names`.

Reply:
Yeah, that's more neat. Done